src/stellar/StellarTools.ts

```diff
@@ -81,11 +87,33 @@ export class StellarTools implements EdgeCurrencyTools {
     return { stellarKey: keypair.secret() }
   }
 
-  async importPrivateKey(privateKey: string): Promise<{ stellarKey: string }> {
-    privateKey.replace(/ /g, '')
-    stellarApi.Keypair.fromSecret(privateKey)
-    if (privateKey.length !== 56) throw new Error('Private key wrong length')
-    return await Promise.resolve({ stellarKey: privateKey })
+  async importPrivateKey(userInput: string): Promise<StellarPrivateKeys> {
+    if (/^(0x)?[0-9a-fA-F]{64}$/.test(userInput)) {
+      // It looks like a private key, so validate the hex:
+      userInput.replace(/ /g, '')
+      stellarApi.Keypair.fromSecret(userInput)
+      if (userInput.length !== 56) throw new Error('Private key wrong length')
+
+      return await Promise.resolve({
+        stellarKey: userInput,
+        stellarMnemonic: ''
```

Comment:
Don't save this as an empty string. The cleaner says it can be undefined so don't save it at all